src/system-fetch.js

```diff
@@ -59,6 +59,29 @@
       }
     };
   }
+  else if (typeof fetch != 'undefined') {
```

Comment:
I'm not saying it should be restricted to service workers, I'm just saying we shouldn't default to using it in Node, where we want to use `fs` not HTTP for loading.